http-gateway/web/package.json

```diff
@@ -6,6 +6,14 @@
     "@auth0/auth0-react": "^1.8.0",
     "@babel/cli": "^7.16.0",
     "@craco/craco": "^6.4.3",
+    "@opentelemetry/api": "^1.1.0",
+    "@opentelemetry/context-zone": "^1.3.1",
+    "@opentelemetry/instrumentation": "^0.29.2",
+    "@opentelemetry/instrumentation-fetch": "^0.29.2",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.29.2",
+    "@opentelemetry/propagator-b3": "^1.3.1",
```

Comment:
This propagator is not currently supported by http-server - you can see it in header as `b3`:
![image](https://user-images.githubusercontent.com/23374974/175933930-975b1790-e592-4152-9f89-160202c70ba3.png)


Pls, could you replace it with core W3C propagator which is implemented in core ? https://npm.io/package/@opentelemetry/core